### PR TITLE
ska3-matlab 2020.8

### LINKED ISSUE #418: Update sot/xija to Version 4.19.0
Includes the following merges:
- PR #90: Enhance StepFunctionPower component to allow multiple instances
- PR #86: Fix bug in component param set attribute

### LINKED ISSUE #419: Release ska3-matlab 2020.8
# Packages to include

- xija 4.19.0

# Checklist

- [x] Document all changes
- [x] Build all packages
   - [x] noarch
   - [x] linux 64
   - [x] mac 64
- [x] Copy Packages to test channel
- [x] Document test status
- [x] For ska3-matlab: request permission and install in GRETA matlab[32]/test
- [x] Get approval
- [x] Move individual packages to main conda channel
- [x] Update production systems:
   - [x] announce to aca@cfa
   - [x] move ska3-flight package to main conda channel
   - [x] update HEAD ska3/flight as aca user
   - [x] run ska_testr on HEAD
   - [x] update ska3/[flight|matlab][32] on cheru as sot user
   - [x] update ska3/[flight|matlab][32] on chimchim as SOT user
   - [x] announce to chandracode
   - [ ] For ska3-flight: announce to mpweekly

### PR DESCRIPTION
**Pull request by javierggt**
# ska3-matlab 2020.8

## Summary:

The ska3-matlab 2020.8 release includes the latest  xija release 4.19.0. 

# Code Changes

## ska3-matlab (2020.5 -> 2020.8rc1)

**sot/xija**: 4.18.1 -> 4.19.0 (4.18.1 -> 4.19.0)
- PR #90: Enhance StepFunctionPower component to allow multiple instances
- PR #86: Fix bug in component param set attribute

# Testing status

The test failures are the same as in the previous ska3-matlab release, which were considered acceptable:

   * Ska.engarchive test failure is a known and non-impacting issue with the sync process / sync test.
   * Ska.quatutil failure is due to a know and non-impacting test defect.
   * maude failure is due to an acceptable regression caused by the recent MAUDE data updates

## ska3-matlab (2020.5 -> 2020.8rc1)
test_spec_SKA3_HEAD on kady

```
**************************************************************
***      Package                  Script            Status ***
*** ------------------ ---------------------------- ------ ***
***   Chandra.Maneuver                 test_unit.py   pass ***
***   Chandra.Maneuver           post_check_logs.py   pass ***
***       Chandra.Time                 test_unit.py   pass ***
***       Chandra.Time           post_check_logs.py   pass ***
*** Chandra.cmd_states                 test_unit.py   pass ***
*** Chandra.cmd_states           post_check_logs.py   pass ***
***         Quaternion                 test_unit.py   pass ***
***         Quaternion           post_check_logs.py   pass ***
***            Ska.DBI                 test_unit.py   pass ***
***            Ska.DBI           post_check_logs.py   pass ***
***          Ska.Numpy                 test_unit.py   pass ***
***          Ska.Numpy           post_check_logs.py   pass ***
***        Ska.ParseCM             test_unit_git.sh   pass ***
***        Ska.ParseCM           post_check_logs.py   FAIL ***
***          Ska.Shell                 test_unit.py   pass ***
***          Ska.Shell           post_check_logs.py   pass ***
***          Ska.Table             test_unit_git.sh   ---- ***
***          Ska.Table           post_check_logs.py   ---- ***
***     Ska.engarchive    test_make_archive_long.sh   ---- ***
***     Ska.engarchive                 test_unit.py   FAIL ***
***     Ska.engarchive           post_check_logs.py   FAIL ***
***            Ska.ftp                 test_unit.py   pass ***
***            Ska.ftp           post_check_logs.py   pass ***
***       Ska.quatutil             test_unit_git.sh   pass ***
***       Ska.quatutil           post_check_logs.py   pass ***
***            Ska.tdb                 test_unit.py   pass ***
***            Ska.tdb           post_check_logs.py   pass ***
***          acis_taco                 test_unit.py   pass ***
***          acis_taco           post_check_logs.py   pass ***
***       acisfp_check              test_regress.sh   pass ***
***       acisfp_check         test_regress_head.sh   pass ***
***       acisfp_check           post_check_logs.py   pass ***
***       acisfp_check              post_regress.py   pass ***
***       acisfp_check         post_regress_head.py   pass ***
***              agasc                 test_unit.py   pass ***
***              agasc           post_check_logs.py   pass ***
***              annie                 test_unit.py   pass ***
***              annie           post_check_logs.py   pass ***
***        chandra_aca                 test_unit.py   pass ***
***        chandra_aca           post_check_logs.py   pass ***
***            cxotime                 test_unit.py   pass ***
***            cxotime           post_check_logs.py   pass ***
***          dea_check              test_regress.sh   pass ***
***          dea_check         test_regress_head.sh   pass ***
***          dea_check           post_check_logs.py   pass ***
***          dea_check              post_regress.py   pass ***
***          dea_check         post_regress_head.py   pass ***
***          dpa_check              test_regress.sh   pass ***
***          dpa_check         test_regress_head.sh   pass ***
***          dpa_check           post_check_logs.py   pass ***
***          dpa_check              post_regress.py   pass ***
***          dpa_check         post_regress_head.py   pass ***
***               kadi         test_regress_long.sh   ---- ***
***               kadi                 test_unit.py   pass ***
***               kadi           post_check_logs.py   pass ***
***               kadi         post_regress_long.py   ---- ***
***              maude                 test_unit.py   FAIL ***
***              maude           post_check_logs.py   FAIL ***
***               mica                 test_unit.py   pass ***
***               mica           post_check_logs.py   pass ***
***   package_manifest              test_regress.sh   pass ***
***   package_manifest           post_check_logs.py   pass ***
***   package_manifest              post_regress.py   pass ***
***           parse_cm                 test_unit.py   pass ***
***           parse_cm           post_check_logs.py   pass ***
***            proseco                 test_unit.py   pass ***
***            proseco           post_check_logs.py   pass ***
***         psmc_check              test_regress.sh   pass ***
***         psmc_check         test_regress_head.sh   pass ***
***         psmc_check           post_check_logs.py   pass ***
***         psmc_check              post_regress.py   pass ***
***         psmc_check         post_regress_head.py   pass ***
***             pyyaks                 test_unit.py   pass ***
***             pyyaks           post_check_logs.py   pass ***
***           sparkles                 test_unit.py   pass ***
***           sparkles           post_check_logs.py   pass ***
***          starcheck              test_regress.sh   pass ***
***          starcheck           post_check_logs.py   pass ***
***          starcheck              post_regress.py   pass ***
***          timelines test_unit_git_sybase_long.sh   ---- ***
***          timelines           post_check_logs.py   ---- ***
***               xija                 test_unit.py   pass ***
***               xija           post_check_logs.py   pass ***
**************************************************************

```
# Fixes Issues

Fixes #418
Fixes #419

### CODE REVIEW

File: pkg_defs/ska3-flight/meta.yaml [hunk 1]
```diff
@@ -60,4 +60,4 @@ requirements:
     - starcheck ==13.6.0
     - tables3_api ==0.1.1
     - testr ==4.3.1
-    - xija ==4.18.2
+    - xija ==4.19.0
```
Comment (jeanconn) [inline pkg_defs/ska3-flight/meta.yaml:63]: I think we decided not to update flight for now.

Comment (jeanconn) [reply pkg_defs/ska3-flight/meta.yaml]: And you beat me apparently pushing more commits at the instant of making the pr...

Comment (javierggt) [reply pkg_defs/ska3-flight/meta.yaml]: yes, I was changing it when you were writing the comment :-)

**Comment by javierggt** [comment]
Merging this PR to proceed with shiny. If the current tests are ok, the current candidate will be tagged as the release and we are done. If not, then we well continue on this branch.

**Pull request merged**